doc/classes/Basis.xml

```diff
@@ -64,7 +64,7 @@
 			<return type="Vector3">
 			</return>
 			<description>
-				Assuming that the matrix is a proper rotation matrix (orthonormal matrix with determinant +1), return Euler angles (in the YXZ convention: first Z, then X, and Y last). Returned vector contains the rotation angles in the format (X angle, Y angle, Z angle).
+				Returns the basis' Euler angles (in the YXZ convention: first Z, then X, and Y last). Returned vector contains the rotation angles in the format (X angle, Y angle, Z angle). See [method get_rotation_quat] if you need a quaternion instead.
```

Comment:
This implies that the basis just "has" Euler angles, but in reality it has a rotation matrix which is then converted. I think it would make more sense as "Return the rotation of this basis in the form of Euler angles". Same thing below.

Reply:
@Calinou ^